public/src/components/channelManagement/epicTests/epicTestEditor.tsx

```diff
@@ -107,7 +107,10 @@ const EpicTestEditor: React.FC<EpicTestEditorProps> = ({
 
   const onVariantDelete = (deletedVariantName: string): void => {
     const updatedVariantList = test.variants.filter(variant => variant.name !== deletedVariantName);
-    onVariantsChange(updatedVariantList);
+    const controlProportionSettings =
+      updatedVariantList.length >= 2 ? test.controlProportionSettings : undefined;
+
+    updateTest({ ...test, variants: updatedVariantList, controlProportionSettings });
```

Comment:
I considered updating `onVariantsChange` and adding the `controlProportionSettings` logic there. However, it's only on deleting variants that we need to do this, so it seemed to make sense to just update here (but does mean we need to use `updateTest` here instead of `onVariantsChange`)